js/main.js

```diff
@@ -139,18 +145,58 @@ function generateWeatherContent(response) {
     $mainHeading.className = 'heading hidden';
     $weatherHeading.textContent = weatherCondition + '  ' + cityTemp + ' \u00B0F';
     $headingContainer.appendChild($weatherHeading);
+    if (weatherCondition === 'Clear') {
+      weatherCondition = 'Sun';
+    }
     getArtWeather(weatherCondition);
     $weatherContainer.className = 'weather-container';
     $weatherPage.appendChild($weatherContainer);
+
     var $cityName = document.createElement('h1');
     $cityName.textContent = response.name;
-    $weatherContainer.append($cityName);
-    var newCityTemp = document.createElement('div');
+    $cityName.className = 'row column-full center';
+    $weatherContainer.prepend($cityName);
+    var $moreText = document.createElement('h3');
+    $moreText.className = 'row column-full center';
+    $moreText.textContent = 'Current Weather Conditions:';
+    $weatherContainer.appendChild($moreText);
+    var $weatherRowContainer = document.createElement('div');
+    $weatherRowContainer.className = 'row center';
+    $weatherContainer.append($weatherRowContainer);
+    var $iconContainer = document.createElement('div');
+    $iconContainer.className = 'column-half';
+    $weatherRowContainer.append($iconContainer);
+    var $weatherIcon = document.createElement('i');
+    $iconContainer.append($weatherIcon);
+    var $conditionsContainer = document.createElement('div');
+    $conditionsContainer.className = 'column-half';
+    $weatherRowContainer.append($conditionsContainer);
+    var newCityTemp = document.createElement('h2');
     newCityTemp.textContent = cityTemp + ' \u00B0F';
-    $weatherContainer.appendChild(newCityTemp);
-    var newCityWeather = document.createElement('div');
+    $conditionsContainer.appendChild(newCityTemp);
+    var newCityWeather = document.createElement('h2');
     newCityWeather.textContent = weatherCondition;
-    $weatherContainer.appendChild(newCityWeather);
+    $conditionsContainer.appendChild(newCityWeather);
+    if (weatherCondition === 'Clouds') {
+      $headerColor.className = 'header clouds';
+      $weatherIcon.className = 'fas fa-cloud fa-7x';
+      $weatherContainer.style.color = 'white';
+    } else if (weatherCondition === 'Sun') {
+      $headerColor.className = 'header clear';
+      $weatherIcon.className = 'fas fa-sun fa-7x';
+      $weatherContainer.style.color = 'rgb(255, 227, 70)';
+      newCityWeather.textContent = 'Clear';
```

Comment:
But here, if the value is "Sun", you are using the term "clear". Is there a reason this value is being changed?